infrastructure/aws/lambda/handler.py

```diff
@@ -48,6 +35,6 @@ async def startup_event() -> None:
     loop.run_until_complete(app.router.startup())
 
 
-def lambda_handler(event: Dict[str, Any], context: Any) -> Dict[str, Any]:
+def lambda_handler(event: LambdaEvent, context: LambdaContext) -> dict[str, Any]:
```

Comment:
FYI, this is actually redundant. This function can be eliminated, and the `handler` variable can simply be renamed to `lambda_handler`.